build.sbt

```diff
@@ -724,7 +724,6 @@ lazy val Dependencies = new {
 
   object Decline {
     val core = Def.setting("com.monovore" %%% "decline-effect" % "2.3.0")
```

Comment:
this is still pointing to decline-effect

Reply:
that is probably a copy/paste error. ill fix